Makefile

```diff
@@ -1,16 +1,22 @@
-all: elastic_search_up index_documents give_indexing_some_time rest_api_up run_query
+all: elastic_search_up give_es_some_time populate_indexes rest_api_up give_api_some_time run_query run_faq_query run_rulebook_query
 
 elastic_search_up:
 	docker-compose up -d elasticsearch
 
-index_documents:
-	python document_indexing/populate_document_store.py
+give_es_some_time:
+	sleep 30
 
-give_indexing_some_time:
-	sleep 20
+populate_indexes:
+	python document_indexing/populate_document_store.py
 
 rest_api_up:
 	docker-compose up -d haystack-api
 
-run_query:
-	 curl -H 'Content-Type: application/json' -H 'Accept: application/json' -d '{"query": "What is the objective of the game?"}' http://127.0.0.1:8000/query
+give_api_some_time:
+	sleep 120
+
+run_faq_query:
```

Comment:
I am getting error when running this, is this running for you? If so, I will have a better look on why it does not work for me

Reply:
`File "/home/user/haystack/pipelines/base.py", line 2105, in _validate_node_names_in_params
    f"No node(s) or global parameter(s) named {', '.join(invalid_keys)} found in pipeline."
ValueError: No node(s) or global parameter(s) named FaqRetriever, CustomClassifier found in pipeline.
`

Reply:
Hm, did you remember to run `docker-compose build` in your repository directory? The docker images need to be updated after every change in code